charts/nutanix-csi-storage/templates/ntnx-csi-controller-deployment.yaml

```diff
@@ -2,27 +2,44 @@
 #
 # example usage: kubectl create -f <this_file>
 
-kind: StatefulSet
+kind: Deployment
 apiVersion: apps/v1
 metadata:
-  name: csi-provisioner-ntnx-plugin
+  name: nutanix-csi-controller
```

Comment:
Why are we changing the names ? Will helm upgrade handle cleanup of old resources ?

Reply:
I change name to follow a more standard and understandable logic
yes helm take care of cleaning